Source/Runtime/Measurement Server.lvproj

```diff
@@ -14,6 +14,7 @@
 		<Property Name="specify.custom.address" Type="Bool">false</Property>
 		<Item Name="MeasurementLink Measurement Server" Type="Folder">
 			<Item Name="MeasurementLink Measurement Server.lvlib" Type="Library" URL="../MeasurementLink Measurement Server/MeasurementLink Measurement Server.lvlib"/>
+			<Item Name="NI MeasurementLink Session Management Client.lvlib" Type="Library" URL="../NI MeasurementLink Session Management Client/NI MeasurementLink Session Management Client.lvlib"/>
```

Comment:
I won't argue the name since I can't come up with a better alternative.  :)

Reply:
I don't know why, but tacking "NI" to the front of the library name when nothing else has it really bothers me. I would suggest just dropping "Client" from the library name for now until we have a better plan in place to fix all of the naming inconsistencies. i.e. `MeasurementLink Session Management.lvlib`.

Reply:
That sounds better for now. Implemented.